code/modules/atmospherics/machinery/other/meter.dm

```diff
@@ -51,7 +51,9 @@
 
 /obj/machinery/meter/proc/reattach_to_layer()
```

Comment:
can meters be covered? or are they so minor we don't need to care

Reply:
theyre rare enough that i dont think it matters enough to make them nullspaced